File: src-scala-orig/test/scala/cern/colt/matrix/tint/impl/DenseColumnIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseColumnIntMatrix2DViewTest(arg0: String) extends DenseColumnIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseColumnIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseColumnIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseColumnIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/DenseColumnIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class DenseColumnIntMatrix2DViewTest extends DenseColumnIntMatrix2DTest {

    public DenseColumnIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseColumnIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseColumnIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseColumnIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
